pkgs/development/tools/hobbes/default.nix

```diff
@@ -0,0 +1,38 @@
+{ stdenv, fetchFromGitHub, cmake, gcc6, llvm_6, ncurses, python27, readline, zlib }:
+
+stdenv.mkDerivation {
+  name = "hobbes";
+  version = "unstable-2020-02-18";
+
+  src = fetchFromGitHub {
+    owner = "morgan-stanley";
+    repo = "hobbes";
+    rev = "8f920f320223f301a3e732e478a2facafd94687a";
+    sha256 = "0av6wp3gsjnil2j8xfn3rkx2r50sfcaq19d7a8bpnfhq2z8q67kg";
+  };
+
+  nativeBuildInputs = [
```

Comment:
I don't know this project, but `ncurses`, `readline`, and `zlib` should most likely be in `buildInputs`. From a quick glance at their `CMakeList.txt`, `llvm_6` should probably also be in`buildInputs`. If Python is only used during testing (as it appears), it should be in `checkInputs`. Also, did you try if this works with Python 3? Python 2 is EOL.

Reply:
Pushed changes reflecting your suggestions (and squashed the other commits). It does work with Python 3. Does it look ok?

Reply:
Did you try to put Python in `checkInputs`? Looks ok otherwise. Can't do a proper review though, since I can't test this package.

Reply:
Just did that. Indeed Python is only used to test the implementation of the fregion script.